README.md

```diff
@@ -11,6 +11,14 @@ The following dependnecies need to be installed for all of the scripts to operat
 - jenkspy: https://pypi.org/project/jenkspy/
 - numpy : https://pypi.org/project/numpy/
 
+### Downloading Requirements
+
+All the dependencies for all the scripts related to the project can be installed via `pip install`:
+
+```bash
+pip install -r requirements.txt
```

Comment:
Has the `requirements.txt file` been added?

Reply:
Yeah. I can commit it. I was thinking to either create a separate pull request for that or add it here.

Reply:
Well, since you already made the changes to the `README.md` for it we might as well include it here.